packages/router/src/grpc/custody/authorize.ts

```diff
@@ -1,4 +1,4 @@
-import { GrpcRequest } from '@penumbra-zone/transport';
+import { GrpcRequest } from '@penumbra-zone/transport'; // old types
```

Comment:
these annotations note where some package is trying to interface with the old transport. most of it is just message types.